README.md

```diff
@@ -48,7 +48,8 @@ terraform help
 mkdir segment-datalakes-tf
 ```
 * Create `main.tf` file
-    * Update the `segment_sources` variable in the `locals` to the sources you want to sync
+    * Update the `external_ids` variable in the `locals` to the workspace ID. This will allow all sources in the workspace to be synced to the Data Lake without any extra setup.
+      * **Note - Existing users** may be using the `sourceID` here instead of the `workspaceID`, which was the previous configuration. Setting this value as the `sourceID` is still supported for existing users for backwards compatibility. Follow instruction [here](#Using-workspaceID-as-externalID) to migrate to `workspaceID`. This will ensure you do not need to update this value for each source you want to add.
```

Comment:
the `#Using-workspaceID-as-externalID` link doesn't seem to be working in the preview view

Reply:
Yeah ill fix this in a follow-up update once I merge and get the link to the section.